delegatable_credentials/src/msbm/sps_eq_uc_sig.rs

```diff
@@ -264,17 +264,17 @@ impl<E: Pairing> Signature<E> {
         ),
         DelegationError,
     > {
-        if update_key.start_index > insert_at_index {
+        if update_key.start_index as usize > insert_at_index {
```

Comment:
Update type of `user_public_key` and `max_attributes_per_commitment` in `Signature::new*` accordingly.